docs/source/workflows/profiler.md

```diff
@@ -64,9 +64,9 @@ Native integrations with `aiq eval` to allow for running of the profiler through
 ## Using the Profiler
 
 ### Step 1: Enabling Instrumentation on a Workflow [Optional]
-**NOTE:** If you don't set it, AIQ toolkit will inspect your code to infer frameworks used. We recommend you set it explicitly.
+**NOTE:** If you don't set it, NeMo Agent toolkit will inspect your code to infer frameworks used. We recommend you set it explicitly.
 To enable profiling on a workflow, you need to wrap the workflow with the profiler decorators. The decorators can be applied to any workflow using the `framework_wrappers` argument of the `register_function` decorator.
-Simply specify which AIQ toolkit supported frameworks you will be using anywhere in your workflow (including tools) upon registration and AIQ toolkit will automatically apply the appropriate profiling decorators at build time.
+Simply specify which NeMo Agent toolkit supported frameworks you will be using anywhere in your workflow (including tools) upon registration and NeMo Agent toolkit will automatically apply the appropriate profiling decorators at build time.
```

Comment:
I'm not really sure here, but we should avoid using "Nemo Agent toolkit" more than once in a sentence.

```suggestion
Simply specify which NeMo Agent toolkit supported frameworks you will be using anywhere in your workflow (including tools) upon registration and the toolkit will automatically apply the appropriate profiling decorators at build time.
```